src/main/kotlin/org/jetbrains/research/testspark/tools/TestProcessor.kt

```diff
@@ -140,7 +140,7 @@ class TestProcessor(
 
         CommandLineRunner.run(command as ArrayList<String>)
 
-        return testExecutionError
+        return if (testExecutionResult.first == 0) "" else testExecutionResult.second
```

Comment:
Once you replace a pair with a data class, I recommend creating a method `succeeded/executionSucceeded(): Bool` and call it here instead of `== 0`. 

We do not want a `TestProcessor` to manually check the success condition; it is not his responsibility to know cli command-related concepts.